app/decorators/user_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class UserDecorator < Draper::Decorator
```

Comment:
Elegant solution :heart: I think I would have called it `AnonymizedUserDecorator`, because not all decoration should be in one class. What do you think?